WalletWasabi/BitcoinCore/CoreNode.cs

```diff
@@ -91,7 +92,7 @@ public static async Task<CoreNode> CreateAsync(CoreNodeParams coreNodeParams, Ca
 				coreNode.RpcEndPoint = rpce;
 
 				var rpcClient = new RPCClient($"{authString}", coreNode.RpcEndPoint.ToString(coreNode.Network.DefaultPort), coreNode.Network);
-				coreNode.RpcClient = new RpcClientBase(rpcClient);
+				coreNode.RpcClient = new CachedRpcClient(rpcClient, new MemoryCache(new MemoryCacheOptions()));
```

Comment:
How about add prevention to overuse the memory with `SizeLimit`? 
https://docs.microsoft.com/en-us/aspnet/core/performance/caching/memory?view=aspnetcore-3.1#use-setsize-size-and-sizelimit-to-limit-cache-size

Reply:
Done.